### ISSUE TITLE
Internet Explorer setting descriptions are forwarding to Bing

### ISSUE THREAD

**Issue by Naxterra** [MicrosoftDocs/memdocs#3477]
As title says, Internet Explorer policies are forwarding to Bing search page without any results.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 8f0f4d5d-fdd1-22e7-6372-9916b199209f
* Version Independent ID: caeb9f8b-30ad-7f02-4740-56522b2f9b1b
* Content: [Settings list for the Windows 10/11 MDM security baselines in Microsoft Intune - Microsoft Intune](https://learn.microsoft.com/en-us/mem/intune/protect/security-baseline-settings-mdm-all)
* Content Source: [memdocs/intune/protect/security-baseline-settings-mdm-all.md](https://github.com/MicrosoftDocs/memdocs/blob/main/memdocs/intune/protect/security-baseline-settings-mdm-all.md)
* Service: **microsoft-intune**
* Sub-service: **protect**
* GitHub Login: @Brenduns
* Microsoft Alias: **brenduns**

**Comment by yogkumgit** [comment]
@Brenduns Please help us in resolving this issue. Thanks

**Comment by Brenduns** [comment]
@yogkumgit   This information has been passed on to the product group to have them fix the underlying FWLINKS.  

**What’s up:** 
For each setting, the content publishes the same *Learn more* link as the link is provided/presented in the Intune policy UI.   Each link uses a unique FWLINK and ID, which for affected settings also points to Search instead of specific setting information.  Because the UI links are not correct, the same links that are then copied to and presented in content  will remain broken until the UI link destinations are fixed.  Those fixes are on the back end to redirect a FWLINK and ID to a specific location.  (FWLINKS are used to link out so that as information changes, the link can be edited to point to a new location, without having to edit product code to change that link destination.)

So in short,  once the product team resolves each affected link to the correct content in the Internet Explorer CSP docs, the content will also reflect that correct link.  

With this information having been passed to the product team, this issue can be closed. There is no direct fix to be delivered in docs. (Won’t fix – product bug). That fix will be automatically delivered once the product team can correct the destinations for each of the broken links.  There is no eta for this action.

**Comment by yogkumgit** [comment]
@Naxterra Hope this comment is helpful for you. If you see a documentation update is required, please feel free to open an issue for the same. We proceed here to close it. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.
